includes/class-wcmp-postcode-fields.php

```diff
@@ -581,22 +581,10 @@ public function merge_street_number_suffix($order_id): void
         $billingHasCustomAddressFields  = self::isCountryWithSplitAddressFields($_POST['billing_country']);
         $shippingHasCustomAddressFields = self::isCountryWithSplitAddressFields($_POST['shipping_country']);
 
-        if (version_compare(WOOCOMMERCE_VERSION, '2.1', '<=')) {
-            // old versions use 'shiptobilling'
-            $shipToDifferentAddress = ! isset($_POST['shiptobilling']);
-        } else {
-            // WC2.1
-            $shipToDifferentAddress = isset($_POST['ship_to_different_address']);
-        }
+        $shipToDifferentAddress = isset($_POST['ship_to_different_address']);
```

Comment:
kun je alle $_POST stuff gewoon een keer sanitizen? dan zijn we ook van die meldingen af